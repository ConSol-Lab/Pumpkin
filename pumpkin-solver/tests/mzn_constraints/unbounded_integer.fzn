var int: x:: output_var;
var -2..2: y:: output_var;
var -2..2: z:: output_var;

constraint int_le(z,x);
constraint int_le(x,y);

solve satisfy;
