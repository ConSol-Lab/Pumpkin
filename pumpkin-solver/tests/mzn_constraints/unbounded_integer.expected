x = -2;
y = -2;
z = -2;
----------
x = -2;
y = -1;
z = -2;
----------
x = -2;
y = 0;
z = -2;
----------
x = -2;
y = 1;
z = -2;
----------
x = -2;
y = 2;
z = -2;
----------
x = -1;
y = -1;
z = -2;
----------
x = -1;
y = 0;
z = -2;
----------
x = -1;
y = 1;
z = -2;
----------
x = -1;
y = 2;
z = -2;
----------
x = -1;
y = -1;
z = -1;
----------
x = -1;
y = 0;
z = -1;
----------
x = -1;
y = 1;
z = -1;
----------
x = -1;
y = 2;
z = -1;
----------
x = 0;
y = 0;
z = -2;
----------
x = 0;
y = 1;
z = -2;
----------
x = 0;
y = 2;
z = -2;
----------
x = 0;
y = 0;
z = -1;
----------
x = 0;
y = 1;
z = -1;
----------
x = 0;
y = 2;
z = -1;
----------
x = 0;
y = 0;
z = 0;
----------
x = 0;
y = 1;
z = 0;
----------
x = 0;
y = 2;
z = 0;
----------
x = 1;
y = 1;
z = -2;
----------
x = 1;
y = 2;
z = -2;
----------
x = 1;
y = 1;
z = -1;
----------
x = 1;
y = 2;
z = -1;
----------
x = 1;
y = 1;
z = 0;
----------
x = 1;
y = 2;
z = 0;
----------
x = 1;
y = 1;
z = 1;
----------
x = 1;
y = 2;
z = 1;
----------
x = 2;
y = 2;
z = -2;
----------
x = 2;
y = 2;
z = -1;
----------
x = 2;
y = 2;
z = 0;
----------
x = 2;
y = 2;
z = 1;
----------
x = 2;
y = 2;
z = 2;
----------
==========
